src/content/containers/Home.js

```diff
@@ -33,6 +33,27 @@ type InstallButtonProps = {
   className?: string,
 };
 
+type UploadButtonProps = {
+  retrieveProfileFromFile: File => void,
+};
+
+class UploadButton extends PureComponent {
+  props: UploadButtonProps;
+
+  render() {
+    return (
+      <div>
+        <input type="file" ref="file"></input>
```

Comment:
nit: eslint isn't happy with your use of double quotes: please change them to single quotes :)